pkg/util/utils.go

```diff
@@ -99,6 +99,26 @@ type JobRunResult struct {
 	Failed             bool     `json:"failed"`
 	HasUnknownFailures bool     `json:"hasUnknownFailures"`
 	Succeeded          bool     `json:"succeeded"`
+
+	// SetupStatus can be "", "Success", "Failure"
+	SetupStatus      string          `json:"setupStatus"`
+	InstallOperators []OperatorState `json:"installOperators"`
+	UpgradeOperators []OperatorState `json:"upgradeOperators"`
+}
+
+const (
+	InfrastructureTestName = `[sig-sippy] infrastructure should work`
+	InstallTestName        = `[sig-sippy] install should work`
+	UpgradeTestName        = `[sig-sippy] upgrade should work`
+
+	Success = "Success"
+	Failure = "Failure"
+)
+
+type OperatorState struct {
+	Name string `json:"name"`
+	// OperatorState can be "", "Success", "Failure"
+	State string `json:"state"`
```

Comment:
was there a compelling reason not to use a boolean here?  anticipating future expansion of the list of states?

Reply:
in kube, we found nothing is a bool.  You can imagine this developing into subdivision of failures for reporting of naughty operators.  It could have caught storage degraded versus something not available.  and I think registry is showing progressing in one :(

Reply:
but you're only going to have the boolean info of "test failed" or "test succeeded" so unless you introduce specific tests for things like "is operator foo progressing" and then report state=progressing based on that test success/failure, i'm not sure how you'll expose the operator state through this.

anyway i'm not going to hold it up